lib/inferno/entities/test_kit.rb

```diff
@@ -0,0 +1,133 @@
+require_relative '../repositories/test_kits'
+require_relative '../repositories/test_suites'
+
+module Inferno
+  module Entities
+    # @example
+    #
+    # module USCoreTestKit
+    #   class TestKit < Inferno::Entities::TestKit
+    #     id :us_core
+    #     title 'US Core Test Kit'
+    #     description <<~DESCRIPTION
+    #       This is a big markdown description of the test kit.
+    #     DESCRIPTION
+    #     suite_ids ['us_core_v311', 'us_core_v400', 'us_core_v501', 'us_core_v610']
+    #     tags ['SMART App Launch', 'US Core']
+    #     last_updated '2024-03-07'
+    #     version '0.6.4'
+    #     maturity 'High'
+    #     authors ['Author One', 'Author Two']
+    #     repo 'https://github.com/inferno-framework/us-core-test-kit'
+    #   end
```

Comment:
one other idea i had was for maybe documenting extra services / tools loaded into the test kit at this level (e.g. reference-server).  Which, yeah, has to be well synchronized with docker-compose and nginx etc, but i'd think is where the information should belong.  Perhaps we just kick that down the road a bit, as it would involve lots more thought.

Reply:
Yeah, I don't think we should take that on yet.